### PR TITLE
Create tutorial for creating a blog using Syna

### PR DESCRIPTION
**Pull request by mpourismaiel**
**Merge after merging #6**

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -64,7 +64,7 @@ lastmod = ["lastmod", ":git", "date"]
   weight = 10
 
 [[menu.main]]
-  url = "/about"
+  url = "/syna"
```
Comment (stp-ip) [inline config.toml]: Shouldn't this be about.okkur.org/syna or syna.okkur.org?

File: content/blog/2019-10-29-hello-world/content.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+weight = 100
+
+title = "Hello World!"
+title_align = "left"
+
+[sidebar]
+  active = true
++++
+
+This is the first blog post of the Okkur Labs.
```
Comment (stp-ip) [inline content/blog/2019-10-29-hello-world/content.md]: I'll add some content here once we merge and deploy this in :)

File: content/blog/_global/contact.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
++++
```
Comment (stp-ip) [inline content/blog/_global/contact.md]: Do we need a contact form?

**Review by stp-ip — CHANGES_REQUESTED** [review]
A few comments, but sounds good overall. Looking forward to use merging this in.

File: content/blog/_global/contact.md (hunk 3 above)
Comment (mpourismaiel) [reply content/blog/_global/contact.md]: It can be used as a comment form. That is if we create an email for it that can accept spam :D

Comment (stp-ip) [reply content/blog/_global/contact.md]: Ok let's go without this for now ;)

File: config.toml (hunk 1 above)
Comment (stp-ip) [reply config.toml]: ?

**Review by stp-ip — APPROVED** [review]

**Pull request merged**